JustRideSDKWrapper/include/dummy.h

```diff
@@ -0,0 +1,5 @@
+/*
+ * Copyright 2023 Masabi Ltd
+ */
+
+// Swift Package Manager needs at least one source file.
```

Comment:
Please change the opening part of this comment to: "A Swift Package Manager target needs..."